lib/plugins/aws/package/compile/events/kafka.js

```diff
@@ -243,6 +254,10 @@ class AwsCompileKafkaEvents {
           };
         }
 
+        if (startingPositionTimestamp !== undefined) {
```

Comment:
We should treat `null` in the same way. So let's update the condition to `!= null`

Reply:
yep, sorry I did m previous commit from my mobile and I did not realised these 2 guys where still not consistent